docs/car-park-skills.md

```diff
@@ -59,54 +56,121 @@ aea add skill fetchai/carpark_client:0.1.0
 aea install
 ```
 
-Second, add the ledger info to the aea config (`car_data_buyer/aea-config.yaml`):
-``` bash
-ledger_apis:
-  fetchai:
-    network: testnet
-```
-
 Alternatively to the previous two steps, simply run:
 ``` bash
 aea fetch fetchai/car_data_buyer:0.1.0
 cd car_data_buyer
 aea install
 ```
 
-### Generate wealth for the car data buyer AEA
+Additionally, create the private key for the car data buyer AEA based on the network you want to transact.
 
-Add a private key to the car data buyer AEA:
-``` bash
+To generate and add a key for Fetch.ai use:
+```bash
 aea generate-key fetchai
 aea add-key fetchai fet_private_key.txt
 ```
 
-To fund the car data buyer AEA on test-net you need to request some test tokens from a faucet.
+To generate and add a key for Ethereum use:
+```bash
+aea generate-key ethereum
+aea add-key ethereum eth_private_key.txt
+```
 
-First, print the address:
```

Comment:
The update configs step is only relevant when the agent was not fetched.

Reply:
It is sections. It is all about creating the AEAs. Then you have to update the configs based on the network you want to connect to. Even if you fetch the AEA it is going to be connected to the default network. The user must know how to change the network, from my point of view at least.